packages/chat-core-aws-connect/src/CoreAwsConnectProvider.ts

```diff
@@ -0,0 +1,11 @@
+import { ChatCoreAwsConnectImpl } from "./infra/ChatCoreAwsConnectImpl";
+import { ChatCoreAwsConnect } from "./models";
+
+/**
+ * Provider for the ChatCore integration with AWS Connect.
+ *
+ * @public
+ */
+export function provideChatCoreAwsConnect(): ChatCoreAwsConnect {
```

Comment:
perhaps the provide function can accept a config to configure logger for the chat session

Reply:
Good idea! Added